docs/book/src/tasks/kubeadm-control-plane.md

```diff
@@ -0,0 +1,46 @@
+# Kubeadm control plane
+
+Using the Kubeadm control plane type to manage a control plane provides several ways to upgrade control plane machines.
+
+## Upgrading workload clusters
+
+The high level steps to fully upgrading a workload cluster are to first upgrade the control plane and then upgrade
+the worker machines.
+
+### Upgrading the control plane machines
+
+#### How to upgrade the underlying machine image
+
+To upgrade the control plane machines underlying machine images, the `MachineTemplate` resource referenced by the
+`KubeadmControlPlane` must be changed. Since `MachineTemplate` resources are immutable, the recommended approach is to
+
+1. Copy the existing `MachineTemplate`.
+2. Modify the values that need changing, such as instance type or image ID.
+3. Create the new `MachineTemplate` on the management cluster.
+4. Modify the existing `KubeadmControlPlane` resource to reference the new `MachineTemplate` resource.
+
+The final step will trigger a rolling update of the control plane using the new values found in the `MachineTemplate`.
+
+#### How to upgrade the Kubernetes control plane version
+
+To upgrade the Kubernetes control plane version, which will likely, depending on the provider, also upgrade the
+underlying machine image, make a modification to the `KubeadmControlPlane` resource's `Version` field. This will trigger
```

Comment:
nit:
```suggestion
underlying machine image, make a modification to the `KubeadmControlPlane` resource's `Spec.Version` field. This will trigger
```